web_session_management/__manifest__.py

```diff
@@ -12,4 +12,5 @@
         "wizards/res_users_sessions_wizard.xml",
         "views/http_session_user.xml",
     ],
+    "installable": False,
```

Comment:
There was an incompatibility there, so I make it uninstallable, as we are not really using it :wink:. Another option would be making an include and exclude...

Reply:
@albariera Done, can you check the changes please¿

Reply:
Done, thank you for the review. LGTM :+1: